packages/react-composites/tests/browser/app/index.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+
+import { ChatAdapter, createAzureCommunicationChatAdapter, ChatComposite } from '../../../src';
```

Comment:
something in the path or filename should indicate this is a chat app

Reply:
Done